bundles/com.espressif.idf.ui/plugin.xml

```diff
@@ -100,11 +95,6 @@
 			id="com.espressif.idf.ui.command.eclipeguide"
 			name="%command.name.9">
 		</command>
-		<command
-			defaultHandler="com.espressif.idf.ui.handlers.UpdateEspIdfHandler"
-			id="com.espressif.idf.ui.command.update"
-			name="%command.name.updateEspIdfMaster">
-		</command>
```

Comment:
We also need to think this can be supported from eim? How user could update their existing master branch?